Gulpfile.js

```diff
@@ -588,7 +589,14 @@ gulp.task("VerifyLKG", /*help*/ false, [], () => {
 gulp.task("LKGInternal", /*help*/ false, ["lib", "local"]);
 
 gulp.task("LKG", "Makes a new LKG out of the built js files", ["clean", "dontUseDebugMode"], () => {
-    return runSequence("LKGInternal", "VerifyLKG");
+    const lib = "./lib";
+    const sizeBefore = getDirSize(lib);
+    const seq = runSequence("LKGInternal", "VerifyLKG");
+    const sizeAfter = getDirSize(lib);
+    if (sizeAfter > (sizeBefore * 1.10)) {
+        throw new Error("The lib folder increased by 10% or more. This likely indicates a bug.");

```

Comment:
How do we want to confirm "no, this really is what we meant to do"?

Reply:
u can manually check in the LKG. this happens after the LKG has been built and updated.